index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>Document</title>
+</head>
+
+<body>
+
+    <section class="first-section">
+        <h1 class="section-heading">Welcome to our website</h1>
+        <p class="section-heading">This is a <b>Sample Webpage</b>, a HTML and CSS template designed by the <a
+                class="link" href="https://www.youtube.com/" target="_blank"> GeeksforGeeks Team</a>. The photos in this
+            template are
+            designed by our </p>
+        <p class="section-heading"><strong>Graphic Design Team</strong></strong>. This template is designed to explain
+            the basis of HTML and CSS in our first Web Design course . </p>
+        <p class="section-title">Section 1</p>
+    </section>
+    <section class="second-section">
+        <p class="section-title">Section 2</p>
+        <div class="main-section">
+            <div class="section-part">
+                <h2>GFG IDE</h2>
+                <p>
+                    An integrated devellopment environment to run and test your codes in almost all of the popular and
```

Comment:
Fix the spelling mistakes for the whole file.
CC: @nb-1231

Reply:
resolved